### PR TITLE
Intro improvements

### PR DESCRIPTION
**Pull request by nrbrook**
Suggested intro improvements based on Charles’ answers to my questions. I added estimated timeframes and price points, these were really just guesses and could be refined. I removed the team list as that would be difficult to maintain going forwards.

### CODE REVIEW

**Comment by nrbrook** [comment]
It would be good to include some sketches of what the designs might look like – I could have a go but it isn't my forte, if there are any sketches or CAD already done then it would be better to use those.

File: README.md [hunk 1]
```diff
@@ -1,45 +1,71 @@
...
 
-Existing companies produce electronic stethoscopes, though none natively broadcast to Bluetooth headphones. No stethoscope includes pulse oximetry.
+This project is developing rapidly and constantly evolving. We are starting with the simple amplifier + speaker, then moving to the Bluetooth design. All designs are intended to be constructed by anyone with some basic electronics knowledge and a soldering iron, and possibly access to a 3D printer.
 
-ThinkLabs https://www.thinklabs.com/
-3M/Littman https://www.littmann.com/3M/en_US/littmann-stethoscopes/products/~/3M-Littmann-Electronic-Stethoscope-Model-3200/?N=5932256+8711017+3293188392&rt=rud
+Costs to build are very rough estimates. Our goal is to 
+
+### "Quick-and-dirty" speaker puck (~10USD, prototypes expected mid-April)
```
Comment (nrbrook) [inline README.md]: @charleslucas Do you think this estimate is reasonable?

File: README.md [hunk 2]
```diff
@@ -1,45 +1,71 @@
...
+
+The simplest possible design doesn’t use a headset at all - simply a single unit with a cup/microphone, a AA-battery-powered amplifier, and a speaker. We’re currently in the prototyping stage for this using off-the-shelf electronic components that anyone should be able to order online.  If possible, we would like to use existing containers - reusing standard pill bottles for instance - to avoid having to 3D-print a housing.
+
+Sterilization is a factor as well.  If possible, using something like a pill bottle would provide a housing that can be easily sterilized.  If not possible, we can resort to putting units in ziploc bags that can be sterilized between patients.  
+
+We still have some basic questions to answer to prove this solution will work - we have to be able to reliably reproduce the sounds that a sick patients’ lungs make.  We are doing research to develop an audio test library, but ultimately won’t be certain until we can get some units into the hands of professionals for testing.
+
+### "Smarter" speaker puck (~20USD, prototypes expected late April - mid May)
```
Comment (nrbrook) [inline README.md:39]: @charleslucas Do you think this estimate is reasonable?

File: README.md [hunk 3]
```diff
@@ -1,45 +1,71 @@
...
+
+One step up from there is to replace our simple analog electronics with some digital logic and a microprocessor.  We’re looking at the ESP32, which is common and relatively cheap, and there are a number of good development platforms on the market.  It comes with integrated bluetooth and wifi (which we would use for the next model), as well as lithium-ion battery charging logic.  This would allow us to potentially do some digital signal processing of the sound, to clear up noise or amplify certain frequencies.
+
+This will build on the knowledge we gather about microphones and speakers and housings, and will likely require some software development.  It will take longer, so is the second stage solution.
+
+### Smart Bluetooth puck (~30USD, prototypes expected May-June)
```
Comment (nrbrook) [inline README.md:45]: @charleslucas Do you think this estimate is reasonable?

**Review by charleslucas — APPROVED** [review]
This looks great!  I may re-word the time estimates - I think we are in the "prototyping" stage now and making good progress, but given that it's already mid-April, I'm guessing we'll have some "prototypes" - i.e. something that could reasonably be built by other people according to our specs for testing - in early May.

**Pull request merged**